test/remediation.test.ts

```diff
@@ -0,0 +1,45 @@
+import * as fs from 'fs';
+import * as path from 'path';
+import { test } from 'tap';
+import { getUpgrades } from '../src/lib/snyk-to-html';
+
+const admZipVulnPath = path.join(__dirname, 'fixtures', 'adm-zip.vuln.json');
+
+test('getUpgrades with empty data', async (t) => {
+  // Arrange
+  const upgrade = {};
+  const vulnerabilities = {};
+  const expected = [];
+  // Act
+  const result = getUpgrades(upgrade, vulnerabilities);
+  // Assert
+  t.same(result, expected, 'should return empty array');
+});
+
+test('getUpgrades with data', async (t) => {
+  // Arrange
+  const upgrade = {
+    "adm-zip@0.4.7": {
+      "upgradeTo": "adm-zip@0.4.11",
+      "upgrades": [
+        "adm-zip@0.4.7"
+      ],
+      "vulns": [
+        "npm:adm-zip:20180415"
+      ],
+    },
+  };
+  const vulnerabilities = [JSON.parse(fs.readFileSync(admZipVulnPath, 'utf-8'))];
+  const expected = [{
+    upgradeTo: 'adm-zip@0.4.11',
+    upgradeFrom: 'adm-zip@0.4.7',
+    vulns: [{
+      severity: 'high',
+      title: 'Arbitrary File Write via Archive Extraction (Zip Slip)',
+    }],
+  }];
+  // Act
+  const result = getUpgrades(upgrade, vulnerabilities);
+  // Assert
+  t.same(result, expected, 'should return expected array');
+});
```

Comment:
lets add some more tests for getSeverityScore and getHighestSeverity

Reply:
you'll have to export them https://github.com/snyk/snyk-to-html/pull/80/files#diff-710ba31a7c97c48c7c0ce5a21bdd1548R73 so this test can import

Reply:
Totally agree, on it :)